Gems/ROS2/Code/Source/Camera/CameraSensorConfiguration.h

```diff
@@ -18,8 +18,18 @@ namespace ROS2
     {
         AZ_TYPE_INFO(CameraSensorConfiguration, "{386A2640-442B-473D-BC2A-665D049D7EF5}");
         static void Reflect(AZ::ReflectContext* context);
-
+        struct Limits
+        {
+            int m_minWidth;
+            int m_minHeight;
+        };
+        static constexpr Limits m_limits{
+            1,
+            1,
+        };
```

Comment:
Declaring a struct `Limits` here seems like overkill. Can't `m_minWidth` and `m_minHeight` be constexpr values for the `CameraSensorConfiguration` instead?